poller.js

```diff
@@ -3,21 +3,34 @@ import axios from "axios"
 
 dotenv.config({ path: './server/.env' });
 
-const POLLING_INTERVAL_MS = 30000;
+const POLLING_INTERVAL_MS = 10000;
```

Comment:
I don't think this is necessary (10s interval), because the go server caches data for 2 minutes.  It won't hurt anything, but you're going to get the same data most of the time, so 30 seconds is probably okay.